experiments/torch-train-test-debug-template.ipynb

```diff
@@ -345,7 +345,7 @@
     "    train_loss = 0.0\n",
     "    val_loss = 0.0\n",
     "    for key, loader in tqdm(dataloaders.items(), desc=\"Datasets\", leave=False):\n",
-    "        mtype_class = import_object(f\"kinoml.core.measurements.{key}\")\n",
```

Comment:
Add an if statement to switch between kinase specific models and kinome wide models.

Reply:
@schallerdavid , this workaround does the trick! The PR is ready to be merged!